helpers/app

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Install other YunoHost apps
+#
+# usage: ynh_install_apps --apps="a_ynh b_ynh?domain=DOMAIN&path=PATH&admin=USER&language=fr&is_public=1&password=pass&port=666"
+# | arg: -a, --apps=     - apps to install
+#
+# Requires YunoHost version *.*.* or higher.
+ynh_install_apps() {
+	# Declare an array to define the options of this helper.
+	local legacy_args=a
+	local -A args_array=([a]=apps=)
+	local apps
+	# Manage arguments with getopts
+	ynh_handle_getopts_args "$@"
+
+	# Split the list of apps in an array
+	local apps_list=($(echo $apps | tr " " "\n"))
+	
+	# For each app
+	for i in "${apps_list[@]}"
+	do
+		# Retrieve the name of the app (part before _ynh)
+		local oneapp=$(echo "$i" | awk -F'_ynh' '{print $1}')
+		[ -z "$oneapp" ] && ynh_die --message="You didn't provided a YunoHost app to install"
+
+		# Retrieve the arguments of the app (part after ?)
+		local oneargument=$(echo "$i" | awk -F'?' '{print $2}')
+		[ ! -z "$oneargument" ] && oneargument="--args \"$oneargument\""
+
+		if ! yunohost app list | grep -q "$oneapp"
```

Comment:
Ideally it is more robust to use `yunohost app list --output-as json | jq ...` with the appropriate jq expression / filter :stuck_out_tongue_winking_eye:  (not trivial but it's worth learning to craft such stuff). There may be similar examples in the other helpers to get inspiration from

Reply:
found it:
`yunohost app list --output-as json --quiet | jq -e --arg id $oneapp '.apps[] | select(.id == $id)'`